lib/elixir/src/elixir_tokenizer.erl

```diff
@@ -1479,7 +1479,11 @@ invalid_do_error(Prefix) ->
   "    end\n\n"
   "or the equivalent construct:\n\n"
   "    if(some_condition?, do: :this, else: :that)\n\n"
-  "where \"some_condition?\" is the first argument and the second argument is a keyword list"}.
+  "where \"some_condition?\" is the first argument and the second argument is a keyword list.\n\n"
+  "** Note:\n\n"
```

Comment:
I love this message, but would not use the `** Note:` here. I'd just write the rest of the message 🙃

Reply:
👍 works for me!